components/ui/AppLink.vue

```diff
@@ -1,13 +1,17 @@
 <template>
   <component
-    :is="isInternal(url) ? 'nuxt-link' : 'a'"
+    :is="!isStatic && isInternal(url) ? 'nuxt-link' : 'a'"
     :href="url"
-    :to="isInternal(url) ? url : null"
+    :to="!isStatic && isInternal(url) ? url : null"
     :rel="isExternal(url) ? 'noopener' : null"
     :target="isExternal(url) ? '_blank' : null"
+    :title="title"
     @click="segment && $trackClickEvent(segment)"
```

Comment:
Non-blocking: when this was a menu, we wanted all the menus to have the option of tracking the click, now this is more general, I think we could get rid of it and move the responsibility to the consumer.